ui/nuxeo-page-provider-display-behavior.js

```diff
@@ -145,7 +145,7 @@ export const PageProviderDisplayBehavior = [
       maxItems: {
         type: Function,
```

Comment:
Since `maxItems` is holding a number, this should be:
```suggestion
        type: Number,
```

Reply:
👍